rest/aggs/aggs.go

```diff
@@ -0,0 +1,42 @@
+package aggs
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/polygon-io/client-golang/rest/client"
+)
+
+// todo: add comments for godoc
+
+type Client struct {
+	client.BaseClient
+}
+
+func (ac *Client) Get(ctx context.Context, params GetParams, opts ...client.Option) (*AggsResponse, error) {
+	res := &AggsResponse{}
+	path := "/v2/aggs/ticker/{ticker}/range/{multiplier}/{resolution}/{from}/{to}"
+	err := ac.Call(http.MethodGet, path, params, res, append([]client.Option{client.WithContext(ctx)}, opts...)...)
```

Comment:
Instead of having to do this context to options dance in a bunch of places should the Call method instead take a context as the first argument?

Reply:
makes sense! just updated it